src/k8-client/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 edition = "2021"
 name = "k8-client"
-version = "13.1.1"
+version = "13.2.0"
```

Comment:
should bump this to `14.0.0` as fluvio future is bump up

Reply:
fixed